tests/test_git.py

```diff
@@ -758,6 +835,15 @@ def test_setup_cmd_parser(self):
         self.assertEqual(parsed_args.git_path, '/tmp/gitpath')
         self.assertEqual(parsed_args.uri, 'http://example.com/')
         self.assertEqual(parsed_args.branches, ['master', 'testing'])
+        self.assertFalse(parsed_args.no_update)
```

Comment:
I would put the code below in a new tests. It's hard to understand why this is failing. It's failing because `--no-update` and `--latest-items` are mutually exclusive. Having this in a test named like `test_mutual_exclusive_update` or something like that will make live easier. Someone would read the test and will know what's going on.